src/coreclr/vm/peimage.h

```diff
@@ -119,9 +119,9 @@ class PEImage final
     static PTR_PEImage OpenImage(
         LPCWSTR pPath,
         MDInternalImportFlags flags = MDInternalImport_Default,
-        BundleFileLocation bundleFileLocation = BundleFileLocation::Invalid());
+        ProbeExtensionResult probeExtensionResult = ProbeExtensionResult::Invalid());
 
-    static PTR_PEImage FindByPath(LPCWSTR pPath, BOOL isInBundle);
+    static PTR_PEImage FindByPath(LPCWSTR pPath, BOOL isInBundle, BOOL isExternalData);
```

Comment:
Can I assume that all combinations of `isInBundle` and `isExternalData` are valid?

Reply:
They should not both be true - added an assert.